Bottom: bdd29612fc1bc30c1ea761ded034e727283d4506
Top:    85a8e7996d7b4684dd98fa212b7bc7ebcb2cd35d
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-07-23 19:24:23 +0100

Refresh of crosshair-c-don-t-snap-to-an-e

---

diff --git a/src/crosshair.c b/src/crosshair.c
index d8752e7..67613b6 100644
--- a/src/crosshair.c
+++ b/src/crosshair.c
@@ -862,7 +862,7 @@ FitCrosshairIntoGrid (LocationType X, LocationType Y)
   snap_data.y = y;
 
   ans = NO_TYPE;
-  if (TEST_FLAG (SNAPPINFLAG, PCB))
+  if (!PCB->RatDraw && TEST_FLAG (SNAPPINFLAG, PCB))
     ans = SearchScreenGridSlop (Crosshair.X, Crosshair.Y,
                                 ELEMENT_TYPE, &ptr1, &ptr2, &ptr3);
